src/window.rs

```diff
@@ -105,50 +121,66 @@ impl WindowWrapper {
         let previous_size = LogicalSize::new(&window).unwrap();
         let previous_dpis = dpis(&window).unwrap();
 
-        WindowWrapper { 
-            context, window, skulpin_renderer, renderer,
+        WindowWrapper {
+            context,
+            window: Some(window),
+            skulpin_renderer,
+            renderer,
             mouse_down: false,
             mouse_position: LogicalSize {
                 width: 0,
-                height: 0
+                height: 0,
             },
             title: String::from("Neovide"),
             previous_size,
             previous_dpis,
             ignore_text_input: false,
             transparency: 1.0,
-            fullscreen: false
+            fullscreen: false,
         }
     }
 
     pub fn synchronize_settings(&mut self) {
-        let editor_title = { EDITOR.lock().title.clone() };
-        if self.title != editor_title {
-            self.title = editor_title;
-            self.window.set_title(&self.title).expect("Could not set title");
-        }
+        if let Some(mut window) = self.window.take() {
+            let editor_title = { EDITOR.lock().title.clone() };
+            if self.title != editor_title {
+                self.title = editor_title;
+                window.set_title(&self.title).expect("Could not set title");
+            }
 
-        let transparency = { SETTINGS.get::<WindowSettings>().transparency };
-        if let Ok(opacity) = self.window.opacity() {
-            if opacity != transparency {
-                self.window.set_opacity(transparency).ok();
-                self.transparency = transparency;
+            let transparency = { SETTINGS.get::<WindowSettings>().transparency };
+            if let Ok(opacity) = window.opacity() {
+                if opacity != transparency {
+                    window.set_opacity(transparency).ok();
+                    self.transparency = transparency;
+                }
             }
-        }
 
-        let fullscreen = { SETTINGS.get::<WindowSettings>().fullscreen };
-        if self.fullscreen != fullscreen {
-            let state = match fullscreen {
-                true => FullscreenType::Desktop,
-                false => FullscreenType::Off
-            };
-            self.window.set_fullscreen(state).ok();
-            self.fullscreen = fullscreen;
+            let fullscreen = { SETTINGS.get::<WindowSettings>().fullscreen };
+            if self.fullscreen != fullscreen {
+                let state = match fullscreen {
+                    true => FullscreenType::Desktop,
+                    false => FullscreenType::Off,
+                };
+                window.set_fullscreen(state).ok();
+                self.fullscreen = fullscreen;
+            }
+            self.window = Some(window);
```

Comment:
I tried to look for a more elegant solution to retain the window, but it looks like this is the only way to do it.